### PR TITLE
init release-1.3

### PR DESCRIPTION
**Pull request by zhu733756**
Signed-off-by: zhu733756 <talonzhu@yunify.com>

Features:
- add CHANGLOG.md
- add RELEASE.md

### CODE REVIEW

File: CHANGLOG.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Now Every receiver can set a template for itself. @wanjunlei
+- Add an API to send notifications directly. @wanjunlei
+
+### CHANGE
+- Upgrade the crd version to v2beta2.
+
+## v1.3.0 / 2021-08-10
+
+### FEATURES
+- Add support to send notifications to SMS platforms Huawei Cloud (#90 #94). @zhu733756
```
Comment (benjaminhuo) [inline CHANGLOG.md:35]: => to Huawei SMS platforms

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## Publish the new release
+
+For new minor and major releases, create the `release-<major>.<minor>` branch starting at the PR merge commit.
+From now on, all work happens on the `release-<major>.<minor>` branch.
+
+Bump the version in the `VERSION` file in the root of the repository.
+Build and push the container image:
+
+```bash
+make build
```
Comment (benjaminhuo) [inline RELEASE.md:56]: If you take a look at the notification manager's makefile, the build and push of nm are a bit different, you can adjust this to :
```
Make sure you've set up docker buildx environment, then run the following command to build and push amd64 and arm64 images at the same time:
make build
```

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: RELEASE.md (hunk 2 above)
Comment (zhu733756) [reply RELEASE.md:58]: fixed

File: CHANGLOG.md (hunk 1 above)
Comment (zhu733756) [reply CHANGLOG.md]: fixed

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**